diff --git a/cmake/VSTConfig.cmake b/cmake/VSTConfig.cmake
index afa2977..2f76a75 100644
--- a/cmake/VSTConfig.cmake
+++ b/cmake/VSTConfig.cmake
@@ -35,6 +35,10 @@ if(NOT VST3_PACKAGE_ARCHITECTURE)
             else()
                 set(VST3_PACKAGE_ARCHITECTURE "i386")
             endif()
+        elseif(VST3_SYSTEM_PROCESSOR MATCHES "^(armv7l)$")
+                set(VST3_PACKAGE_ARCHITECTURE "armv7l")
+        elseif(VST3_SYSTEM_PROCESSOR MATCHES "^(aarch64)$")
+                set(VST3_PACKAGE_ARCHITECTURE "aarch64")
         else()
             message(FATAL_ERROR "We don't know this architecture for VST3: ${VST3_SYSTEM_PROCESSOR}.")
         endif()
